waiter/src/waiter/token.clj

```diff
@@ -213,10 +227,9 @@
                                      (fn [tokens]
                                        (pc/map-from-keys
                                          (fn [token]
-                                           (let [{:strs [deleted] :as token-data} (kv/fetch kv-store token)
+                                           (let [{:strs [deleted last-update-time] :as token-data} (kv/fetch kv-store token)
                                                  token-hash (sd/token-data->token-hash token-data)]
-                                             {:deleted (true? deleted)
-                                              :etag token-hash}))
+                                             (make-index-entry token-hash (true? deleted) last-update-time)))
```

Comment:
`true?` here seems redundant since it's also invoked inside `make-index-entry`. Can we remove it?

Reply:
Removed.